### ISSUE TITLE
Needed regexp example 

### ISSUE THREAD

**Issue by neo1parekh** [tgriesser/checkit#33]
how can I use the regular expression in checkit
as I have to escape single quote using regular expression and have to check.
Expression is (/'/g, "\'");

**Comment by brayn003** [comment]
Hello, any development on this?

**Comment by rhys-vdw** [comment]
Nope. Nor do I remember closing this issue. Feel free to open a PR
